### ISSUE TITLE
Nan loss when training dfn model with voc2012.

### ISSUE THREAD

**Issue by Hanqer** [ycszen/TorchSeg#18]
I use single GPU GTX1080ti with batch-size 4, so I replace DataParallelModel,Reduce, BatchNorm2d with standard nn.BatchNorm2d. Since the pre-trained model can't be downloaded, I trained the model from scratch.
But I got an Nan loss after few iterations, is it not appropriate to modified this?

My torch version is 1.0 with cuda10. Thanks!

**Comment by ycszen** [comment]
Yes, the modification is a little inappropriate.
First, the batch size is too small. In our experiments, the batch size is 32.
Then, if you can use the sync-bn, it is better to use it, which can affect the performance.
Finally, I didn't try to train the DFN from scratch, which, in my opinion, maybe result in low accuracy.
 Besides, I will make the pre-trained models available recently.
